src/importexport/musicxml/internal/musicxml/import/importmusicxmlpass2.cpp

```diff
@@ -6622,6 +6622,23 @@ Note* MusicXmlParserPass2::note(const String& partId,
             } else {
                 headGroup = convertNotehead(noteheadValue);
             }
+        } else if (m_e.name() == "notehead-text") {
+            String noteheadText;
+            while (m_e.readNextStartElement()) {
+                if (m_e.name() == "display-text") {
+                    noteheadText = m_e.readText();
+                } else if (m_e.name() == "accidental-text") {
+                    m_e.skipCurrentElement();
+                } else {
+                    skipLogCurrElem();
+                }
+            }
+            if (noteheadText.size() == 1) {
+                headScheme = (noteheadText == u"H")
+                             ? NoteHeadScheme::HEAD_PITCHNAME_GERMAN : NoteHeadScheme::HEAD_PITCHNAME;
```

Comment:
Shouldn't this get used futher up for NoteFlight, too?

Reply:
What do you mean? I don't understand.

Reply:
A few lines up, in a special case for NoteFlight import, it sets NoteHeadScheme::HEAD_PITCHNAME, not doing the special 'H' checking done here

Reply:
1. Noteflight seems to export non-valid MusicXML, so I don't see amy parallels here.
2. This is just a simple check for that very case. If there's demand, this should be further improved later on anyway. But I don't know of any programs that export it properly yet.